api/repositories/process_repository_test.go

```diff
@@ -451,6 +451,42 @@ var _ = Describe("ProcessRepo", func() {
 		})
 	})
 
+	Describe("GetAppRevisionForProcess", func() {
+		var (
+			appRevision string
+			getErr      error
+		)
+
+		JustBeforeEach(func() {
+			appRevision, getErr = processRepo.GetAppRevisionForProcess(ctx, authInfo, app1GUID, space.Name)
+		})
+		When("the user is not authorized in the space", func() {
```

Comment:
Minor glitch: this `When` can go since it does not have a `BeforeEach`, i.e. brings nothing new to the table. This is our personal preference, some people in the community insist that this way it is more clear, but we tend to opt in for minimalism.

Reply:
done